src/main/kotlin/com/team4099/robot2023/commands/manipulator/IntakeConeCommand.kt

```diff
@@ -0,0 +1,29 @@
+package com.team4099.robot2022.commands.intake
+
+import com.team4099.robot2023.config.constants.ManipulatorConstants
+import com.team4099.robot2023.subsystems.manipulator.Manipulator
+import edu.wpi.first.wpilibj2.command.CommandBase
+import org.littletonrobotics.junction.Logger
+
+class IntakeConeCommand(val manipulator: Manipulator) : CommandBase() {
```

Comment:
for each command: add armState

Reply:
This will later have to use the elevator too right?

Reply:
if we're keeping subsystems separate then it makes sense to schedule this command after elevator has gone to SHELF_HEIGHT or smthing. unless we want to add a precondition to make sure the elevator is at the right height before running this command? but i don't see any particular use case for that

Reply:
yeah, that's fine -- I guess the point really was that there will have to be command groups later that chain these in the right order